mnist/mnist_aws.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<p>Do you want to provide a kubectl command they can run e.g.</p><p> </p><p>kubectl create secret generic aws-secret --from-literal=AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/739/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yes. Change to command line now and this is eaiser.</p>